MEMBERSHIP.md

```diff
@@ -105,7 +120,7 @@ As a project Maintainer, you have the following responsibilities and privileges:
 * You make and approve technical design decisions.
 * You set technical direction and priorities.
 * You define milestones and releases.
-* You mentor and guide contributors to the project, including mentoring and sponsoring potential Triager and Maintainer candidates.
+* You mentor and guide contributors to the project, including mentoring and sponsoring potential Reviewer and Maintainer candidates.
```

Comment:
Reviewer mentors Maintainer candidates?

Reply:
This is listed as a Maintainer responsibility. Existing maintainers mentor reviewer and maintainer candidates.